js/src/util/scrollbar.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * --------------------------------------------------------------------------
+ * Bootstrap (v5.0.0-beta2): util/scrollBar.js
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
+ * --------------------------------------------------------------------------
+ */
+
+import SelectorEngine from '../dom/selector-engine'
+import Manipulator from '../dom/manipulator'
+
+const SELECTOR_FIXED_CONTENT = '.fixed-top, .fixed-bottom, .is-fixed'
+const SELECTOR_STICKY_CONTENT = '.sticky-top'
+
+const getWidth = () => {
```

Comment:
It would be good if you change its name so that it can be understood whose width you are getting :slightly_smiling_face:

Reply:
I am open to any proposal. I have started naming `getScrollbarWidth` etc but is seem too much. We are inside **scrollbar.js** and we need the width of scrollbar.